### ISSUE TITLE
Refused to connect because it violates the following Content Security Policy directive: "default-src 'self'"

### ISSUE THREAD

**Issue by regicsolutions** [axios/axios#4054]
#### Describe the issue
When attempting to make a GET request, i get the following error:

Refused to connect because it violates the following Content Security Policy directive: "default-src 'self'". Note that 'connect-src' was not explicitly set, so 'default-src' is used as a fallback

#### Example Code
Code snippet to illustrate your question

```
  const res = await axios({
    method: 'get',
    url: `https://${allowedHosts}/rest/api/1.0/projects?limit=${pageLimit}`,
    headers: {
              Accept: "application/json",
              'Content-Type': 'application/json',
    }, 
    auth: {
      username,
      password, 
    }
  });
```

#### Expected behavior, if applicable

Request is successful

#### Environment
 - Axios Version [latest]
 - Adapter [e.g. XHR/HTTP]
 - Browser [Chrome]
 - Browser Version [latest]
 - Node.js Version [14]
 - Additional Library Versions [React 17]

#### Additional context/Screenshots
Add any other context about the problem here. If applicable, add screenshots to help explain.
![image](https://user-images.githubusercontent.com/6157895/133174261-170a8f47-fc89-493b-b634-8396bd5d0630.png)

![image](https://user-images.githubusercontent.com/6157895/133174310-4806caff-e661-411b-9bce-0c46e775ad9f.png)

**Comment by keehub** [comment]
How to solve it, brother

**Comment by regicsolutions** [comment]
@keehub my issue was cors related, I ended up using this library to handle cors for me https://www.npmjs.com/package/http-proxy-middleware

**Comment by jasonsaayman** [comment]
Hi 👋 

Please could you retry with the latest version and open a new issue should this error still be relevant?

Thanks

**Comment by adarsh-gupta-2002** [comment]
I'm facing same issue .

**Comment by stlevkov** [comment]
I was able to solve this by enabling the checkbox in the Network tab of the browser:  'disable cache'. Pressed it only once solved the problem.
